### ISSUE TITLE
Annoying indentation

### ISSUE THREAD

**Issue by simonc** [slim-template/ruby-slim.tmbundle#19]
When I write a ruby line with a block le line under it is automatically aligned each time I write something.
For instance, given the following code (the pipe sign is the caret):

``` slim
= content_tag_for :article, article do
  = link_to |, article
```

As soon as I start typing the link title the code is changed like this:

``` slim
= content_tag_for :article, article do
= link_to a|, article
```

It's very annoying since it often causes template errors.
I don't know for sure if it's a bundle issue or a TM one but I only get this problem with slim.

Thanks !

**Comment by tshedor** [comment]
@simonc I believe this was resolved in bf19fa9. Closing this out because it's been a few years and I'm unable to reproduce now. Please feel free to comment if you're able to reproduce.

**Comment by simonc** [comment]
@tshedor Hey :) Yes I think it's been fixed since then. Thanks for following up.
